aries-site/src/examples/templates/list-views/ListImageIdentifierExample.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Box, Image, List, Menu, Text } from 'grommet';
+import { More } from 'grommet-icons';
+
+const data = [
+  {
+    name: 'User 1',
+    description: 'San Jose, CA',
+  },
+  {
+    name: 'User 2',
+    description: 'Fort Collins, CO',
+  },
+  {
+    name: 'User 3',
+    description: 'San Jose, CA',
+  },
+  {
+    name: 'User 4',
+    description: 'Houston, TX',
+  },
+  {
+    name: 'User 5',
+    description: 'Houston, TX',
+  },
+];
+
+export const ListImageIdentifierExample = () => {
+  return (
+    <List
+      data={data}
+      action={() => (
+        <Menu
+          icon={<More />}
+          hoverIndicator
+          items={[{ label: 'Deactivate' }, { label: 'Suspend' }]}
+        />
+      )}
+    >
+      {(datum, index) => (
+        <Box direction="row" gap="small" key={index}>
+          <Box width="xxsmall" height="xxsmall">
+            <Image
+              src="https://images.unsplash.com/photo-1487088678257-3a541e6e3922?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2167&q=80"
```

Comment:
Should the image url be part of the data object? Perhaps tied to the location?

If lists are about scan-ability, then the inclusion of an image should assist the scan-ability. If we display the same image for all, then it is not adding any value and should be removed.

Reply:
Agreed, will put images tied to location.